### PR TITLE
[WIP] Update hacking steps for installing on kubernetes

### PR DESCRIPTION
**Pull request by huikang**
- also remove unused make target: deploy-setup

### CODE REVIEW

**Review by alanconway — APPROVED** [review]
Changes look ok to me but should be reviewed by someone more familiar:

/assign @ewolinetz

File: docs/HACKING.md [hunk 1]
```diff
@@ -6,21 +6,26 @@
...
-    $ kubectl create -f deploy/crd.yaml
+Make sure certificates are pre-generated and deployed as secret, e.g., `make deploy-example-secret`.
 
 Deploy the required roles to the cluster:
 
-    $ kubectl create -f deploy/rbac.yaml
+    $ kubectl create ns openshift-logging
+    $ kubectl apply -f manifests/01-service-account.yaml 
+    $ kubectl apply -f manifests/02-role.yaml 
+    $ kubectl apply -f manifests/03-role-bindings.yaml
```
Comment (ewolinetz) [inline docs/HACKING.md:18]: i actually don't think these files have been kept up to date. i plan on removing them as part of a pr, we can probably remove references to them completely.

**Comment by huikang** [comment]
not working with k8s.

**Pull request closed without merging**